poincare/src/power.cpp

```diff
@@ -127,6 +127,22 @@ bool PowerNode::childAtIndexNeedsUserParentheses(const Expression & child, int c
 
 // Private
 
+template<typename T>
+Complex<T> PowerNode::computeRealRootOfRationalPow(const std::complex<T> c, T p, T q) {
```

Comment:
If I understand well, this method may not find a real root even if it exists (6561^(1/8) = 3.
Explain it and change the name to tryComputeRealRootOfRationalPow?